tests/test.py

```diff
@@ -4350,6 +4351,59 @@ def find_forked_pid(self, filepath):
                     pid = re.search(r'PID: (\d+)', line).group(1)
                     return int(pid)
 
+    def testAcraServerTerminate(self):
+        acra_args = self.get_acra_cli_args({})
+        temp_keystore = self.copy_keystore()
+        config = load_yaml_config('configs/acra-server.yaml')
+        config.update(acra_args)
+        config['keys_dir'] = temp_keystore
+        config['log_to_file'] = tempfile.NamedTemporaryFile().name
+        temp_config = tempfile.NamedTemporaryFile()
+        dump_yaml_config(config, temp_config.name)
+
+        acra = fork(lambda: subprocess.Popen([
+            self.get_acraserver_bin_path(), '--config_file={}'.format(temp_config.name)]))
+        try:
+            self.wait_acraserver_connection(config['incoming_connection_string'])
+        except:
+            stop_process(acra)
+            shutil.rmtree(temp_keystore)
+            raise
+        test_engine = None
+        try:
+            # copied from BaseTestCase._fork_acra
```

Comment:
why do we need to copy `_fork_acra` instead of just use existing acra from the parent BaseTestCase class? for logs we have [mixin](https://github.com/cossacklabs/acra/blob/49459fc1a9d3dbe4dc5634f03bbb6be5e207cd78/tests/test_common.py#L493) that starts acra with writing logs to the files